apps/web/pages/api/email.ts

```diff
@@ -0,0 +1,57 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
was about to ask if the hardcoded info in this endpoint were supposed to be hardcoded until I found the Development part of this in the README. Perhaps we can clarify in the implementation that this endpoint is for testing purposes only, or create another folder to have development-only endpoints

Reply:
We can safely remove it or make it work only on dev environments.

Reply:
also I have my eye on Maizzle to inline tailwind styles later on.